packages/patternfly-4/react-integration/cypress/integration/areachartcolor.spec.ts

```diff
@@ -0,0 +1,15 @@
+describe('Area Chart Color Demo Test', () => {
+  it('Navigate to demo section', () => {
+    cy.visit('http://localhost:3000/');
+    cy.get('#area-chart-custom-colors-demo-nav-item-link').click();
+    cy.url().should('eq', 'http://localhost:3000/area-chart-custom-colors-demo-nav-link');
+  });
+
+  it('Verify area chart', () => {
+    cy.get('.VictoryContainer').should('exist');
+  });
+
+  it('Verify legend string', () => {
+    cy.get('text').contains('Average number of pets');
+  });
```

Comment:
Any chance we can test the height and width of the svg? This would apply to all charts.

Reply:
Added to the most recent commit.